### PR TITLE
[new release] openai (0.0.1)

### PR DESCRIPTION
**Pull request by Nymphium**
OCaml OpenAI binding

- Project page: <a href="https://github.com/Nymphium/openai-ocaml">https://github.com/Nymphium/openai-ocaml</a>

##### CHANGES:

See git log

### CODE REVIEW

**Comment by mseri** [comment]
The tarball seems to have changed:
```
OpamSolution.Fetch_fail("https://github.com/Nymphium/openai-ocaml/archive/refs/tags/0.0.1.tar.gz (Bad checksum, expected sha256=c057bab108ebbaa070e779150dccb7c244f2af8c5a865f9fcd2b75bbc715e1cb)")
```

**Comment by Nymphium** [comment]
@mseri Thank you! I modified.

File: packages/openai/openai.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "OCaml OpenAI binding"
+maintainer: ["Satoru Kawahara <s1311350@gmail.com>"]
+authors: ["Satoru Kawahara <s1311350@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/Nymphium/openai-ocaml"
+bug-reports: "https://github.com/Nymphium/openai-ocaml/issues"
+dev-repo: "git+https://github.com/Nymphium/openai-ocaml.git"
+depends: [
+  "ocaml" {>= "5.0.0"}
```
[the remark this answers is not in the document]
Comment (Nymphium) [reply packages/openai/openai.0.0.1/opam:10]: No, this version doesn't use the feature of the version.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**